CRM/Membershipextrasdata/Factory/PriceField.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class CRM_Membershipextrasdata_Factory_PriceField {
+
+  public static function create($params) {
+    $defaultFieldParams = [
+      "financial_type_id" => "Member Dues",
```

Comment:
Should we check if 'Member Dues' exist? if not then we should create a new financial type id

Reply:
Yes, we should. I updated my PR